index.js

```diff
@@ -270,7 +269,7 @@ module.exports.processV4 = (output, origin, locales = []) => {
     thumbnail,
     audio: formatttedTracks,
     video: video_tracks,
-    silent_video
+    silent_video: formatttedTracks.length === 0
```

Comment:
Too late now...
`formatttedTracks` should be `formattedTracks`

Reply:
I won't bother deploying this but: https://github.com/viviedu/ytdl-process/pull/40